test/test.js

```diff
@@ -10,3 +11,16 @@ fs.readdirSync(path.join(__dirname, 'certs')).forEach(function (file) {
   // x509.parseCert(path.join(__dirname, 'certs', file));
   console.log();
 });
+
+var verified = x509.verify(
+  path.join(__dirname,'certs/enduser-example.com.crt'),
+  path.join(__dirname,'CA_chains/enduser-example.com.chain')
+)
+assert(verified)
```

Comment:
Could you please add a scoping brackets here to avoid the variable `verified` accessible :-)